### PR TITLE
lal_highlight: new crate, release 21.0.0

### PR DESCRIPTION
**Pull request by pmderodat**

### CODE REVIEW

**Comment by mosteo** [comment]
Hi @pmderodat , could you rebase your PR onto the tip of `stable-0.4` so the fixed CI tests can run? Thanks!

**Comment by pmderodat** [comment]
Done, thanks! :-)

File: index/la/lal_highlight/lal_highlight-21.0.0.toml [hunk 1]
```diff
@@ -0,0 +1,22 @@
...
+
+authors = ["AdaCore"]
+maintainers = ["Pierre-Marie de Rodat <pmderodat@kawie.fr>"]
+maintainers-logins = ["pmderodat"]
+licenses = ["GMGPL 3.0"]
+website = "https://github.com/AdaCore/libadalang"
+
+project-files = ["highlight.gpr"]
+executables = ["highlight", "ada2web"]
+
```
Comment (mosteo) [inline index/la/lal_highlight/lal_highlight-21.0.0.toml]: No tags on purpose?

**Review by mosteo — DISMISSED** [review]

File: index/la/lal_highlight/lal_highlight-21.0.0.toml (hunk 1 above)
Comment (pmderodat) [reply index/la/lal_highlight/lal_highlight-21.0.0.toml]: Not on purpose. :-) I just had a quick look at existing tags and it’s not clear to me what tag would be appropriate. Adding a new “libadalang” tag?

Comment (mosteo) [reply index/la/lal_highlight/lal_highlight-21.0.0.toml]: Indeed we have been quite liberal in allowing new tags with submissions, so whatever you think should make your crate pop up.

Comment (pmderodat) [reply index/la/lal_highlight/lal_highlight-21.0.0.toml]: Ok! I’ve added some tags, please let me know if you think they look appropriate, thanks!

Comment (mosteo) [reply index/la/lal_highlight/lal_highlight-21.0.0.toml]: Looks good to me

**Review by mosteo — APPROVED** [review]

**Pull request merged**